_posts/2017-12-31-release-0.16.2.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Release 0.16.2
+---
+
+Just a few weeks after our last release, we are releasing MDAnalysis version
+0.16.2. The [release notes][notes] list the full details of the changes in this
+version. Like the previous one, this release is dedicated to fixing bugs. It
```

Comment:
I wouldn't say it's dedicated to bug fixes, the point of it is the deprecations and we need to be loud about this